src/components/CookieProvider.ts

```diff
@@ -0,0 +1,19 @@
+import { Ajax } from 'src/libs/ajax';
+import { AbortOption } from 'src/libs/ajax/data-provider-common';
+import { azureCookieReadyStore, cookieReadyStore } from 'src/libs/state';
+
+export interface CookieProvider {
+  invalidateCookies: (options?: AbortOption) => Promise<void>;
+}
+
+export const cookieProvider: CookieProvider = {
+  invalidateCookies: async (options: AbortOption = {}) => {
+    const { signal } = options;
+    // TODO: call azure invalidate cookie once endpoint exists, https://broadworkbench.atlassian.net/browse/IA-3498
```

Comment:
is this first TODO not done?  I see the call to an endpoint is in the code here....

Reply:
GCP and Azure have different endpoints: Leo for GCP vs a workspace?-specific proxy URL for Azure. This only calls the Leo endpoint.

Reply:
Yeah, there's a ticket in the IA backlog to do it linked in the comment